app/organization/members/index/template.hbs

```diff
@@ -1,53 +1,36 @@
-{{partial 'organization/members-header'}}
+{{partial 'organization/members-toolbar'}}
 
-<div class="layout-container">
-  <div class="resource-list-grid">
-
-
-    <div class="sort-group organization-members">
-      {{#if sortedInvites}}
-        <h5 class="sort-header">Current Members</h5>
-      {{/if}}
-      <div class="row">
-        {{#each sortedMembers as |user|}}
-          <div class="col-xs-4">
-            {{#aptible-ability user=session.currentUser scope='manage' permittable=organization as |hasAbility|}}
-              {{#if hasAbility}}
-                {{#link-to 'organization.members.edit' user classNames='panel-link'}}
-                  {{partial 'organization/user'}}
-                {{/link-to}}
-              {{else}}
-                {{partial 'organization/user'}}
-              {{/if}}
-            {{/aptible-ability}}
-          </div>
-        {{/each}}
-      </div>
-    </div>
-
-    {{#if sortedInvites}}
-      <div class="sort-group organization-invites">
-        <h5 class="sort-header">Pending Invitations</h5>
-        <div class="row">
-          {{#each sortedInvites as |invitation|}}
-            <div class="col-xs-4">
-              {{partial 'organization/invited-user'}}
-            </div>
-          {{/each}}
-        </div>
-      </div>
+<div class="sort-group organization-members">
+  <ul class='aptable__nav'>
```

Comment:
Can we reuse the nav pills here for consistency?  I agree that we want to revisit those styles, but I think these tabs are a bit too subtle.  See the [app nav tabs](https://github.com/aptible/dashboard.aptible.com/blob/master/app/templates/app/-nav.hbs) as an example